### PR TITLE
[Calendar] add get meetings action

### PR DESCRIPTION
**Pull request by KayMKM**
### Purpose
*What is the context of this pull request? Why is it being done?*
To fix stack overflow issue during searching meeting, add get meetings action

### Changes
*Are there any changes that need to be called out as significant or particularly difficult to grasp? (Include illustrative screenshots for context if applicable.)*

### Tests
*Is this covered by existing tests or new ones? If no, why not?*

### Feature Plan
*Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues.*

#### General
- [ ] I have commented my code, particularly in hard-to-understand areas	
- [ ] I have added or updated the appropriate tests	
- [ ] I have updated related documentation

### CODE REVIEW

File: skills/declarative/calendarskill/extensions/Microsoft.Bot.Solutions.Extensions/Actions/GetMeetings.cs [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+            catch (ServiceException ex)
+            {
+                throw GraphClient.HandleGraphAPIException(ex);
+            }
+
+            var result = new List<MeetingModel>();
+
+            try
+            {
+
```
Comment (DingmaomaoBJTU) [inline skills/declarative/calendarskill/extensions/Microsoft.Bot.Solutions.Extensions/Actions/GetMeetings.cs:72]: Remove extra line

File: skills/declarative/calendarskill/extensions/Microsoft.Bot.Solutions.Extensions/Actions/GetMeetings.cs [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+                        ID = item.Id
+                    };
+
+                    result.Add(meetingModel);
+
+                }
+            }
+            catch(Exception e)
+            {
+                e.ToString();
```
Comment (DingmaomaoBJTU) [inline skills/declarative/calendarskill/extensions/Microsoft.Bot.Solutions.Extensions/Actions/GetMeetings.cs:97]: why not throw?

Comment (KayMKM) [reply skills/declarative/calendarskill/extensions/Microsoft.Bot.Solutions.Extensions/Actions/GetMeetings.cs]: this try catch is useless.. removed.

File: skills/declarative/calendarskill/extensions/Microsoft.Bot.Solutions.Extensions/Actions/GetMeetings.cs (hunk 1 above)
Comment (KayMKM) [reply skills/declarative/calendarskill/extensions/Microsoft.Bot.Solutions.Extensions/Actions/GetMeetings.cs]: this try catch is useless.. removed.

**Pull request merged**